keyboards/gh60/revc/config.h

```diff
@@ -14,14 +14,17 @@ GNU General Public License for more details.
 You should have received a copy of the GNU General Public License
 along with this program. If not, see <http://www.gnu.org/licenses/>.
 */
+
+#pragma once
+
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID    0xFEED
-#define PRODUCT_ID   0x6060
+#define VENDOR_ID    0x4335 // GEEKhack
```

Comment:
Just a general comment:

Not sure how `0x4335` ties to `GEEKhack`. What's the convention here?

Reply:
Yeah, I noticed this a bit late, but the convention I used was just a telephone keypad translation (lol) - G E E K = 4 3 3 5, but we can do 0x4748 if you wish (letters GH to hex)

Reply:
It just caught me off-guard because it deviates from the suggestion used in VIA's docs. It's fine though as long as the VID/PID pairings are unique.